django_mongodb_backend/expressions/search.py

```diff
@@ -168,34 +167,31 @@ def search_operator(self, compiler, connection):
             "path": self.path.as_mql(compiler, connection, as_path=True),
             "query": self.query.value,
         }
-        if self.score is not None:
+        if self.score:
             params["score"] = self.score.as_mql(compiler, connection)
-        if self.fuzzy is not None:
+        if self.fuzzy:
```

Comment:
This should still be `if self.fuzzy is not None` because `fuzzy={}` is valid input